test/test_health.py

```diff
@@ -0,0 +1,155 @@
+import os
```

Comment:
Any reason why this is in `test` and not in `tests` directory?

Reply:
Yes, in `test_health.py` we have `from tests import BaseTestCase` which imports from `/app/tests`. This has a conflict with `/redash-stmo/tests` so this import does not work. Renaming to `test` resolves this. 

Is there perhaps a more elegant way to fix this?

Reply:
Ah right, we add /app to the Python import path in the Dockerfile to bring Redash and redash-stmo together, forgot about that.

I think it should suffice to remove the `__init__.py` file from the /redash-stmo/test directory and rename it to tests since the directory doesn't need to be a Python package (directory with a __init__.py) for pytest to correctly collect the tests. For initialization pytest uses an own practice to put things in a `conftest.py` file. There is a lot of documentation about that [here](https://docs.pytest.org/en/3.7.1/pythonpath.html#pythonpath).